Updated content for Supported Queries (#2382)
couchbase/docs-server: modules/fts/pages/fts-supported-queries-DocID-query.adoc

= DocID Query

A _doc ID_ query returns the indexed document or documents among the specified set.
This is typically used in conjunction queries, to restrict the scope of other queries’ output.

[source,json]
----
{ "ids": [ "hotel_10158", "hotel_10159" ] }
----

> A doc ID Query is demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].

Reviewer: ```suggestion
A demonstration of a doc ID Query using the  Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

Author: Updated
